app.js

```diff
@@ -66,13 +68,19 @@ function displayProgress() {
 
 function addLetterListeners() {
   const letters = document.querySelectorAll('.letter-wrapper');
-  [...letters].forEach(item => item.addEventListener('click', () => letterCheck(item.id)));
+  [...letters].forEach(item => item.addEventListener('click', () => {
+    if (state.correct.includes(item.id) || state.wrong.includes(item.id)) {
+      return;
+    } else {
```

Comment:
nothing will happen after `return` word. 
So, you don't need `else` statement here. 

```
if ([...state.correct, ...state.wrong].includes(item.id)) return;

letterCheck(item.id)
```
`letterCheck` will be invoked only when item.id is not included in `state.correct` or `state.wrong` otherwise we are returning from function. 

I showed you also the rule, that if you have only one expression inside if, you can (but you don't have to) skip `{`